src/scripts/main.js

```diff
@@ -1,3 +1,74 @@
 'use strict';
 
-// write code here
+const container = document.querySelector('.container');
+const table = document.querySelector('tbody');
+const remColBut = document.querySelector('.remove-column');
+const remRowBut = document.querySelector('.remove-row');
+const appendColBut = document.querySelector('.append-column');
+const appendRowBut = document.querySelector('.append-row');
```

Comment:
```suggestion
const appendRowButton = document.querySelector('.append-row');
```

It's not very clear what But means here. Especially if you look not at variable declaration but at variable usage.

Reply:
Thanks.